packages/ra-language-french/src/index.ts

```diff
@@ -117,7 +117,8 @@ const frenchMessages: TranslationMessages = {
                 "Certains changements n'ont pas été enregistrés. Êtes-vous sûr(e) de vouloir quitter cette page ?",
         },
         navigation: {
-            no_results: 'Aucun résultat',
+            clear_filters: 'Effacer les filtres.',
+            no_results: 'Aucun %{resource} trouvé avec les filtres actuels.',
```

Comment:
This won't work well with feminine resources in french

Reply:
```suggestion
            no_results: 'Aucun résultat trouvé avec les filtres actuels.',
```